_episodes/02-REPL.md

```diff
@@ -20,4 +18,56 @@ keypoints:
 
 ---
 
+## Variables
+
+After downloading and executing a julia binary from https://julialang.org Melissa and her classmates face the so called REPL, which stands for <u>r</u>ead-<u>e</u>valuate-<u>p</u>rint-<u>l</u>oop.
+The first thing they try is to perform basic arithmetic operations
+~~~
+1 + 4 * 7.3
+~~~
+{: .language-julia}
+~~~
+30.2
+~~~
+{: .output}
+
+That works as expected.
+It is also possible to bind names to values via the assignment operator `=`, which makes it easier to refer to them later on.
+These names are called _variables_.
+
+~~~
+distance = 30.2
+distance_x_2 = 2 * distance
+~~~
+{: .language-julia}
+~~~
+60.4
+~~~
+{: .output}
+
+Melissa notices that assignment also returns the value.
+
+## Unicode
+
+In julia also unicode characters are allowed as variables like `α = 2`.
+Where unicode characters can be entered by a backslash followed by their LaTeX-name and then pressing <kbd>tab</kbd> (in this case `\alpha`<kbd>tab</kbd>).
```

Comment:
I'd skip this section about Unicode input.  Related, below I'd mention the help-mode in relation to docstring of functions/variable, I think that's more important than finding how to type `∂`

Reply:
I do think it's a good idea to mention docstring lookup. Why do you think the unicode input is unneccessary? I think its quite a selling point for julia.

Reply:
I personally have mixed feeling about Unicode input: it's cool and I use it, but it's also easy to abuse.  For example, I think non-ASCII characters should be used with care in variables or API of functions (but I'm certain I'm guilty of this sin myself).  I'm not sure it's the best feature to present to an audience of novices with diverse background (not all of them may appreciate the usefulness of Unicode characters as identifiers).

Reply:
I share your concerns but also it is quite likely that you will stumble over some exported unicode symbol as a beginner (much more than in any other language, I'd argue), be it Flux or `isapprox` or ... and knowing a way to use it might remove a stepping stone.

Reply:
Fair enough :slightly_smiling_face: